### PR TITLE
(Try to) Fix development build

### LINKED ISSUE #1: Wekan app container fails (it seems permissions)
Following manual finished without errors, but Wekan not start and console shows the following:

```
/home/wekan/.meteor/packages/meteor-tool/.1.3.5_1.1sksnpz++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:165
      throw error;
            ^
Error: EACCES, mkdir '/home/wekan/app/.meteor/local'
```

### PR DESCRIPTION
**Pull request by andresmanelli**
I merged the wekan/wekan Dockerfile into this one and updated the compose file.

I used 8081 because many nodejs apps run on port 3000. If any changes are needed, please let me know.

Tested locally and works well.

Thanks!

(Maybe this fix #1 and #3) ?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,36 @@
-FROM debian:wheezy
+FROM debian:8.9
+MAINTAINER wekan
```
Comment (eemeli) [inline Dockerfile]: The `MAINTAINER` instruction is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

File: Dockerfile [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+ENV METEOR_LOG=debug
 
 RUN \
     # Add non-root user wekan
-    useradd --user-group --system --home-dir /home/wekan wekan && \
+    useradd --user-group --system -m wekan && \
     \
     # OS dependencies
-    apt-get update -y && apt-get install -y --no-install-recommends ${BUILD_DEPS} && \
+    apt-get update -y && apt-get dist-upgrade -y && apt-get install -y --no-install-recommends ${BUILD_DEPS} && \
```
Comment (eemeli) [inline Dockerfile]: Running `dist-upgrade` is against [best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run).

**Review by eemeli — CHANGES_REQUESTED** [review]
Looks good, just needs a few minor fixes.

File: Dockerfile (hunk 1 above)
Comment (andresmanelli) [reply Dockerfile]: Should this be ok?

```diff
-MAINTAINER wekan
+LABEL maintainer="wekan"
```

Comment (andresmanelli) [reply Dockerfile]: @eemeli Updated, let me know if other changes are needed. Thanks!

**Review by eemeli — APPROVED** [review]

**Pull request merged**